README.md

```diff
@@ -1 +1,240 @@
-# arxiv-submission-ui
\ No newline at end of file
+# arXiv Submission UI
+
+This is the primary interface for arXiv users to submit e-prints to arXiv.
+It is comprised of a Flask application built around the [submission core
+events](https://github.com/cul-it/arxiv-submission-core) package.
+
+The Submission UI requires the File Management service.
+
+## Quick start
+
+Submission involves several back-end services, an async worker process, and
+the UI application itself. The easiest way to spin up all of this stuff
+with correct wiring is to use the provided docker-compose configuration.
+
+```bash
+cd /path/to/arxiv-submission-ui
+mkdir /tmp/foo          # Compiler service will use this.
+docker-compose pull     # Pulls in images that you might not have already.
+docker-compose build    # Builds the submission UI
+DIND_SOURCE_ROOT=/tmp/foo CONVERTER_DOCKER_IMAGE=626657773168.dkr.ecr.us-east-1.amazonaws.com/arxiv/converter:0.9 docker-compose up
+```
+
+To get a fresh deployment (e.g. after significant changes to backend stuff),
+you may need to blow away the whole service group. Be sure to use the ``-v``
+flag to drop old volumes.
+
+```bash
+docker-compose rm -v
+```
+
+
+Note! This will start a whole bunch of stuff. Fairly late in the process, a
+bootstrap process will run and generate a bunch of users who are authorized to
+submit things. It will look something like this:
+
+```
+submission-bootstrap     | 1 picoline2058@gmail.com eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJzZXNzaW9uX2lkIjoiYWQyY2UxNmUtMjQwNi00NjgwLWI2NWItMDE3NGYyNDA0MzhlIiwic3RhcnRfdGltZSI6IjIwMTktMDItMTNUMTI6MjM6MzQuMjE2NzcxLTA1OjAwIiwidXNlciI6eyJ1c2VybmFtZSI6InBpY29saW5lMjA1OEBnbWFpbC5jb20iLCJlbWFpbCI6InBpY29saW5lMjA1OEBnbWFpbC5jb20iLCJ1c2VyX2lkIjoxLCJuYW1lIjp7ImZvcmVuYW1lIjoiTWF1cmEiLCJzdXJuYW1lIjoiWmFyZW1iYSIsInN1ZmZpeCI6IlBhbiJ9LCJwcm9maWxlIjp7ImFmZmlsaWF0aW9uIjoiQ29ybmVsbCBVbml2ZXJzaXR5IiwiY291bnRyeSI6InVzIiwicmFuayI6Mywic3VibWlzc2lvbl9ncm91cHMiOlsiZ3JwX3BoeXNpY3MiXSwiZGVmYXVsdF9jYXRlZ29yeSI6ImFzdHJvLXBoLkdBIiwiaG9tZXBhZ2VfdXJsIjoiIiwicmVtZW1iZXJfbWUiOnRydWV9LCJ2ZXJpZmllZCI6ZmFsc2V9LCJjbGllbnQiOm51bGwsImVuZF90aW1lIjoiMjAxOS0wMi0xM1QyMjoyMzozNC4yMTY3NzEtMDU6MDAiLCJhdXRob3JpemF0aW9ucyI6eyJjbGFzc2ljIjowLCJlbmRvcnNlbWVudHMiOlsiKi4qIl0sInNjb3BlcyI6WyJwdWJsaWM6cmVhZCIsInN1Ym1pc3Npb246Y3JlYXRlIiwic3VibWlzc2lvbjp1cGRhdGUiLCJzdWJtaXNzaW9uOnJlYWQiLCJ1cGxvYWQ6cmVhZCIsInVwbG9hZDp1cGRhdGUiLCJ1cGxvYWQ6ZGVsZXRlIiwidXBsb2FkOnJlYWRfbG9ncyJdfSwiaXBfYWRkcmVzcyI6bnVsbCwicmVtb3RlX2hvc3QiOm51bGwsIm5vbmNlIjpudWxsfQ.iNOiCGVIZi5iipElLRyUlnx9uucdK7aytjkvr87FTvI
```

Comment:
Can these be example.org addresses?

Reply:
They could... the idea with using mimesis is we're generating random (hopefully) fake addresses. Is the worry that these might actually exist?

Reply:
I was just being über paranoid on a Monday--not with how mimesis is being used at bootstrap time (totally fine) but how the real-looking examples are being commemorated in the README (and indexed, etc.).

Reply:
Good to be paranoid. Created https://github.com/arXiv/arxiv-submission-ui/issues/92